features/bootstrap/FeatureContext.php

```diff
@@ -14,7 +14,7 @@
 
 define( 'EE', true );
 define( 'EE_VERSION', trim( file_get_contents( EE_ROOT . '/VERSION' ) ) );
-define( 'EE_CONF_ROOT', '/opt/easyengine' );
+define( 'EE_ROOT_DIR', '/opt/easyengine' );
```

Comment:
BTW `EE_CONF_ROOT` is used in almost all repos(ee core, cron, admin-tools, service, auth, site-type-wp, config etc...). So if you're renaming it, just ensure that it is done in all the repos. If you haven't raised PR in these repos raise them and then post them as comment here.

Reply:
@sagarnasit Make changes in other repos when you have time